README.md

```diff
@@ -0,0 +1,52 @@
+# Tech Docs Template
+
+The Tech Docs Template is a static site generator to build technical documentation with a GOV.UK style for GaaP (Government as a Platform) products.
```

Comment:
Ok, I'll update that. I'd like to keep something in about the style as it's the main benefit of this project.